recipe/meta.yaml

```diff
@@ -48,11 +55,11 @@ requirements:
 test:
   commands:
     - $R -e "library('arrow')"           # [not win]
-    - "\"%R%\" -e \"library('arrow')\""  # [win]
+    - "\"%R%\" -e \"library('arrow'); data(mtcars); write_parquet(mtcars, 'test.parquet')\""  # [win]
 
 about:
   home: https://github.com/apache/arrow
-  license: Apache 2.0
+  license: Apache-2.0
   license_file: LICENSE.txt
   summary: R Integration to 'Apache' 'Arrow'.
```

Comment:
FWIW this isn't exactly the Title in `r/DESCRIPTION` (and the version of `DESCRIPTION` commented below is quite stale.

Reply:
The commented version is from the initial recipe creation, we may soon have automated updating from CRAN of these but currently this involves a manual regeneration.